src/foam/flow/Document.js

```diff
@@ -327,33 +336,42 @@ foam.CLASS({
 
         'code': function(code) {
           return function(x) {
-            this.
-              start('pre').
-              cssClass('foam-flow-Document-code').
-              add(code).
-              end();
+            this.start('code').add(code).end();
           };
         },
 
         'foam': function(attributes) {
           return function(x) {
-            // TODO: Reuse FoamTagLoader support, support classloading
```

Comment:
Keep the TODO about reusing FoamTagLoader support as that's about re-using the code in FoamTagLoader, but you can delete the part about classloading :)

Reply:
Done